core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -1288,16 +1288,21 @@ path_get(wasm_exec_env_t exec_env, struct fd_table *curfds,
     size_t expansions = 0;
     char *symlink = NULL;
     size_t symlink_len;
+#ifdef BH_PLATFORM_WINDOWS
+#define PATH_SEPARATORS "/\\"
+#else
+#define PATH_SEPARATORS "/"
+#endif
```

Comment:
Not strictly related to this PR but I realized we'll need it on Windows so paths with backslashes are expanded properly. It would probably be good to verify the behaviour of backslashes with an additional WASI test.